diff --git a/pyuvdata/tests/__init__.py b/pyuvdata/tests/__init__.py
index bb0bd6b..26830b7 100644
--- a/pyuvdata/tests/__init__.py
+++ b/pyuvdata/tests/__init__.py
@@ -80,6 +80,7 @@ def checkWarnings(func, func_args=[], func_kwargs={},
         warnings.simplefilter("always")  # All warnings triggered
         retval = func(*func_args, **func_kwargs)  # Run function
         # Verify
+        w = [w_i for w_i in w if 'dtype size changed' not in str(w_i.message)]
         if len(w) != nwarnings:
             print('wrong number of warnings. Expected number was {nexp}, '
                   'actual number was {nact}.'.format(nexp=nwarnings, nact=len(w)))
